src/GridFieldArchiveAction.php

```diff
@@ -176,7 +177,7 @@ public function handleAction(GridField $gridField, $actionName, $arguments, $dat
                 return;
             }
 
-            if (!$item->canArchive()) {
+            if (!Deprecation::withNoReplacement(fn() => $item->canArchive())) {
```

Comment:
```suggestion
            $canArchive = Deprecation::withNoReplacement(fn() => $item->canArchive());
            if (!canArchive) {
```
Makes this a lot easier to read

Also update all other instances in this PR

Reply:
That's very subjective and I personally don't agree, but I'll make the change to avoid ping pong.

Reply:
Done, fingers cross I caught them all